click-to-reveal/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Click to reveal</title>
+    <link rel="stylesheet" href="./style.css">
+    <script src="./script.js"></script>
+</head>
+
+<body>
+
+    <section id="idt-click-to-reveal">
+        <h1>Click to reveal</h1>
+        <div class="idt-flexBox">
+
+            <div class="idt-sideBox">
+                <button class="idt-btn">
+                    q1
+                </button>
+
+                <button class="idt-btn">
```

Comment:
See comment above. I'd prefer less class names and more CSS selectors.

In this case specifically, since every button you have has the same class, you can just use:

``` css
#idt-click-to-reveal button 
```